packages/example/src/pages/guides/configuration.mdx

```diff
@@ -108,7 +131,7 @@ plugins: [
       resolve: 'gatsby-theme-carbon',
       options: {
 		// will get added to default [300, 400, 600]
-        additionalFontWeights: ['200', '200i]
+        additionalFontWeights: ['200', '200i']
```

Comment:
Can we take this out? It's handled by the variable font now, no need to specify weights